Model/Api/Capabilities.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Omise\Payment\Model\Api;
+
+use OmiseCapabilities;
+
+class Capabilities extends Object
+{
+    private $capabilities;
+
+    public function __construct() {
+        $this->capabilities = OmiseCapabilities::retrieve();
+    }
+
+    public function read() {
```

Comment:
@jacstn also, `read()` method doesn't reflect to its function. Would be nice to having more meaningful method name.